subprojects/docs/src/docs/userguide/jvm/toolchains.adoc

```diff
@@ -344,18 +344,19 @@ In order to disable auto-provisioning, you can use the `org.gradle.java.installa
 [[sec:custom_loc]]
 == Custom Toolchain locations
 
-If auto-detecting local toolchains is not sufficient or disabled, there are additional ways you can let Gradle know about installed toolchains.
+If auto-detecting local toolchains is not sufficient or disabled, there are additional ways you can let Gradle know about installed toolchains. By using one of the following Gradle properties
+in your `gradle.properties` will let Gradle know which directories to look at when detecting JVMs.
```

Comment:
```suggestion
If auto-detecting local toolchains is not sufficient or disabled, there are additional ways you can let Gradle know about installed toolchains.
To let Gradle know which directories to look at when detecting JVMs, use one of the following Gradle properties in `gradle.properties`.
```

Suggestion: lead with the problem, followed by the solution. Trim some unneeded words.